utility/src/main/java/org/oppia/android/util/logging/ConsoleLogger.kt

```diff
@@ -74,7 +74,7 @@ class ConsoleLogger @Inject constructor(
   }
 
   private fun isLogEnable(logLevel: LogLevel): Boolean {
-    return globalLogLevel.logLevel < logLevel.logLevel
+    return globalLogLevel.logLevel <= logLevel.logLevel
```

Comment:
Nice catch. Did the tests help catch this?

Reply:
Yes. The test failed due to this issue